src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
please try to use more descriptive variables - [checklist](https://github.com/mate-academy/js_inverse_robot/blob/master/checklist.md) point 3